axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/EventProcessorInfoConfiguration.java

```diff
@@ -98,4 +98,8 @@ public void initialize(Configuration config) {
             this.config.onStart(Phase.INBOUND_EVENT_CONNECTORS, eventProcessorControlService::get);
         }
     }
+
+    public Component<EventProcessorControlService> getEventProcessorControlService() {
```

Comment:
I wouldn't expose the Component, as that's an internal mechanism for lazily resolving dependencies. As far as I know, none of the other configuration components expose it.